docs/source/examples/creating_models/plot_import_gtfs.py

```diff
@@ -9,20 +26,27 @@
 We use data from Coquimbo, a city in La Serena Metropolitan Area in Chile.
 """
 # %%
-
 # Imports
 from uuid import uuid4
 from os import remove
 from os.path import join
 from tempfile import gettempdir
 
+from pathlib import Path # Check filepaths...
+
 import folium
 import pandas as pd
 from aequilibrae.project.database_connection import database_connection
 
 from aequilibrae.transit import Transit
 from aequilibrae.utils.create_example import create_example
-# sphinx_gallery_thumbnail_path = 'images/plot_import_gtfs.png'
+
```

Comment:
WHy remove the thumbnail?

Reply:
Accidental, reverted changes to this file now